Source/XUnitExtensions/Lit/LitCommandWithRedirection.cs

```diff
@@ -38,6 +38,12 @@ public class LitCommandWithRedirection : ILitCommand {
         errorFile = config.ApplySubstitutions(argumentsList[redirectErrorIndex + 1]).Single();
         argumentsList.RemoveRange(redirectErrorIndex, 2);
       }
+      var redirectErrorAppendIndex = argumentsList.IndexOf("2>>");
+      if (redirectErrorAppendIndex >= 0) {
+        errorFile = config.ApplySubstitutions(argumentsList[redirectErrorAppendIndex + 1]).Single();
```

Comment:
I don't understand this code part, can you please explain it to me?

Reply:
> I get that you want to redirect the standard error to the file as well, but I'm surprised that you have to introduce a new rule for that.

How would it know what `2>>` means without a new rule? It's a from scratch parser for lit commands so it doesn't understand anything by default. Maybe you could make it smart so if it understand `2` and `>>` separately it can also combine them, but it can't do that right now.

Reply:
> How would it know what 2>> means without a new rule? I

I mean, I'm surprised we hadn't had the need tor redirect error output before this PR, that's what I meant.

My comprehension problem was simpler. I just looked at the code above to see what it was doing, and I understand you are generalizing "2>", which I remembered was used in test cases. So that's good for me.